echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/postprocessors/ArtifactPostProcessor.java

```diff
@@ -112,14 +112,31 @@ private JinjaTemplate getTemplateFromProperty(Trigger trigger) {
     }
 
     JinjaTemplate.TemplateType templateType = JinjaTemplate.TemplateType.STANDARD;
-    String customTemplate = (String) properties.get("customFormat");
-    if (Boolean.parseBoolean(customTemplate)) {
+
+    Object customFormat = properties.get("customFormat");
+    if (parseCustomFormat(customFormat)) {
       templateType = JinjaTemplate.TemplateType.CUSTOM;
     }
 
     return jinjaTemplateService.getTemplate(messageFormat, templateType);
   }
 
+  private boolean parseCustomFormat(Object customFormat) {
+    if (customFormat == null) {
+      return false;
+    }
+
+    if (customFormat instanceof Boolean) {
+      return (Boolean) customFormat;
+    }
+
+    if (customFormat instanceof String) {
+      return Boolean.parseBoolean((String) customFormat);
+    }
+
+    throw new RuntimeException("Unexpected customFormat in property file.");
```

Comment:
I would append the `customFormat` object here so the exception reader gets an idea of what they accidentally passed.